plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -339,6 +339,12 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			eventsRule(),
 		},
 	})
+	var statefulSetPVCRule rbacv1.PolicyRule
+	if utilfeature.DefaultFeatureGate.Enabled(features.StatefulSetAutoDeletePVC) {
+		statefulSetPVCRule = rbacv1helpers.NewRule("get", "create", "update", "list").Groups(legacyGroup).Resources("persistentvolumeclaims").RuleOrDie()
```

Comment:
Why does this need `list`?

Reply:
Ah, I think you're right, it's not necessary, we only get.